### ISSUE TITLE
Can't override non-observables in `finnish` with `strict` enabled

### ISSUE THREAD

**Issue by TheKoTech** [cartant/eslint-plugin-rxjs#123]
If `strict` is true, there is no way of allowing $ postfix for non-observables

```js
'rxjs/finnish': [
  'error',
  {
    strict: true,
    names: {
      '^(IS_MOBILE_MODE)': true,
    },
  },
],
```

```ts
// Error: Finnish notation should not be used here.
export const IS_MOBILE_MODE$ = new InjectionToken<Observable<boolean>>('IS_MOBILE_MODE$',  { ... });
```

**Comment by JasonWeinzierl** [comment]
This is fixed in fork `eslint-plugin-rxjs-x` v0.9.0
